pkg/builder/build_client.go

```diff
@@ -13,12 +13,12 @@ import (
 	"github.com/buildbarn/bb-storage/pkg/util"
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/empty"
+	"go.opentelemetry.io/otel/api/trace"
```

Comment:
Nit: These imports should go below the golang.org ones.

Reply:
I've guessed a little at what you mean here.